android-components/components/browser/state/src/test/java/mozilla/components/browser/state/action/TranslationsActionTest.kt

```diff
@@ -203,6 +203,24 @@ class TranslationsActionTest {
         assertEquals(supportedLanguages, tabState().translationsState.supportedLanguages)
     }
 
+    @Test
+    fun `WHEN a SetNeverTranslateSitesAction is dispatched AND successful THEN update neverTranslateSites`() {
```

Comment:
I think I figured out what was going on with the [TranslationsMiddlewareTest.kt](https://github.com/mozilla-mobile/firefox-android/pull/5389/files#diff-44da0a222b637b3152adaf0f82484690dd094aad95c4d1760d7266d4f58a6423) pattern over in #5389. That test now seems to be verifying as expected and I think could be used as a pattern for an additional test on this patch too.

Reply:
Awesome, thanks and thanks for the review! https://github.com/mozilla-mobile/firefox-android/pull/5389 just merged!